nearlib/test/integration.test.js

```diff
@@ -43,6 +49,37 @@ test('create account and then view account returns the created account', async (
     fail('exceeded number of retries for viewing account');
 });
 
+test('deploy contract and make function calls', async () => {
+    var data = [...fs.readFileSync('../tests/hello.wasm')];  
+    const initialAccount = await account.viewAccount(aliceAccountName);
+    var deployResult = await nearjs.deployContract(
+        aliceAccountName,
+        "test_contract",
+        data,
+        "FTEov54o3JFxgnrouLNo2uferbvkU7fHDJvt7ohJNpZY");
+    waitForNonceToIncrease(initialAccount);
+    const args = {
+        "name": "trex"
+    };
+    var viewFunctionResult = await nearjs.callViewFunction(
+        aliceAccountName,
+        "test_contract",
+        "near_func_hello", // this is the function defined in hello.wasm file that we are calling
+        args);
+    expect(viewFunctionResult).toEqual("hello trex");
+});
+
+const waitForNonceToIncrease = async (initialAccount) => {
+    for (i = 0; i < TEST_MAX_RETRIES; i++) {
```

Comment:
nit: this accidentally creates variable in global scope (legacy JS stuff).

you wanna have:

for (__let__ i = 0; i < TEST_MAX_RETRIES; i++) {

I'll look into come automated linter to catch stuff like that.

Reply:
done